pkg/transform/kubernetes/kubernetes.go

```diff
@@ -106,6 +106,16 @@ func (t *Transformer) CreateDeployments(o *object.Service) ([]runtime.Object, er
 		},
 	}
 
+	// Set replica value only if it is given
+	if o.Replicas != nil {
+		// and is positive number
+		if *o.Replicas >= 0 {
+			d.Spec.Replicas = o.Replicas
+		} else {
+			return nil, fmt.Errorf("Replica count is negative in service: %q", o.Name)
+		}
+	}
```

Comment:
If I remove check from here, how do we address the condition where someone wants to use the opencompose as lib?